.tool-versions

```diff
@@ -0,0 +1,2 @@
+elixir 1.13.4
+erlang 24.3.3
```

Comment:
probably best not to include this file in a library

Reply:
I did it intended, but I'm open to remove it

Reply:
i'm concerned that when it ends up in the deps directory for other people it'll force them to install those versions

Reply:
No, this is ignored in the package (like tests and other files)

Reply:
oooooh